solutions/CF Good Subarrays.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: cf-1398C
+source: CF
+title: Good Subarrays
+author: Jesse Choe
+---
+[Official Editorial](https://codeforces.com/blog/entry/81506)
+
+<LanguageSection>
+
+<CPPSection>
+
+## Solution - Prefix Sums + Math
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+using ll = long long;
+using pi = pair<ll, ll>;
+void solve(){
+    int n; cin >> n;
+    vector<int> dp(n+1, 0);
```

Comment:
would be better to replace dp with psum, given it's not really dp.

Reply:
Well most of these editorials use `dp`, so I decided to make all of my psum arrays named `dp` lol.

Reply:
i think it would probably be more clear to name them `psum`...

Reply:
idt it matters lol.